include/constants/battle.h

```diff
@@ -23,12 +23,23 @@
  *   +---------------------------+
  */
 
-#define MAX_BATTLERS_COUNT  4
+enum BattlerPosition
+{
+    B_POSITION_PLAYER_LEFT,
+    B_POSITION_OPPONENT_LEFT,
+    B_POSITION_PLAYER_RIGHT,
+    B_POSITION_OPPONENT_RIGHT,
+    MAX_POSITION_COUNT,
+};
 
-#define B_POSITION_PLAYER_LEFT        0
-#define B_POSITION_OPPONENT_LEFT      1
-#define B_POSITION_PLAYER_RIGHT       2
-#define B_POSITION_OPPONENT_RIGHT     3
+enum BattlerId
+{
+    B_BATTLER_PLAYER_LEFT,
+    B_BATTLER_OPPONENT_LEFT,
+    B_BATTLER_PLAYER_RIGHT,
+    B_BATTLER_OPPONENT_RIGHT,
```

Comment:
I think maybe these should have vague names because battler ID 0 isn't necessarily anything to do with player left, etc?
```suggestion
    B_BATTLER_0,
    B_BATTLER_1,
    B_BATTLER_2,
    B_BATTLER_3,
```
My reasoning is that in the PR we have lines like this:
```gBattlerControllerFuncs[B_BATTLER_OPPONENT_LEFT] = SetControllerToRecordedPlayer;```
Which imo are quite surprising, because it's setting "opponent left" to "player". With the vague names that would be:
```gBattlerControllerFuncs[B_BATLER_1] = SetControllerToRecordedPlayer;```

Reply:
oops, this makes much more sense.

Suggestion applied